tests/lambda_endpoints/test_path_helper.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from sds_data_manager.lambda_code.SDSCode.path_helper import FilenameParser
+
+
+def test_date_checker():
+    """Tests date inputs"""
+    filename = "imap_glows_l0_raw_20231010_20231011_v01-01.pkts"
+    file_parser = FilenameParser(filename)
+    assert file_parser.check_date_input("20200101")
+    assert file_parser.check_date_input("2020-01-01") is False
+    assert file_parser.check_date_input("202301") is False
+    # NOTE: the date is correct but not in the format YYYYMMDD
+    assert file_parser.check_date_input("2023105") is False
+
+
+def test_filename_validator():
```

Comment:
These tests might be easier to parameterize over your assertions.
https://docs.pytest.org/en/7.1.x/example/parametrize.html

```suggestion
@pytest.mark.parametrize("filename,expected", [("imap_glows_l0_raw_20231010_20231011_v01-01.pkts", True), ...]
def test_filename_validator(filename, expected):
    assert FilenameParser(filename).validate_filename() == expected
```